gas/testsuite/gas/riscv/cv-postinc-fail-march.s

```diff
@@ -0,0 +1,26 @@
+# xcorev march command line argument must be specified 
```

Comment:
Maybe highlight here that the march option here can be either xcorev OR xcorevpostinc.
Following the commenting convention in other extensions, this comment could be "# Absence of xcorev or xcorevpostinc march option disables all CORE-V postinc extensions".